release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3518,5 +3517,29 @@ For more information, see xref:../cli_reference/openshift_cli/getting-started-cl
 
 [id="ocp-4-13-23-updating"]
 ==== Updating
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-24"]
+=== RHSA-2023:7475 - {product-title} 4.13.24 bug fix and security update
+
+Issued: 2023-11-29
+
+{product-title} release 4.13.24, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7475[RHSA-2023:7475] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7477[RHSA-2023:7477] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
 
+[source,terminal]
+----
+$ oc adm release info 4.13.24 --pullspecs
+----
+
+[id="ocp-4-13-24-bug-fixes"]
+==== Bug fixes
+
+* Previously, using Cluster Autoscaler with nodes that have CSI storage would cause a `CrashBackoff` loop. With this release, dependencies have been updated to include the nil pointer so the `CrashBackoff` loop will not occur. (link:https://issues.redhat.com/browse/OCPBUGS-23272[*OCPBUGS-23272*])
```

Comment:
i might say "could" cause a CrashBackoff, it's not clear that this happened in every case.

> dependencies have been updated to include the nil pointer so the `CrashBackoff` loop will not occur.

i might say that "dependencies have been updated to improve error handling so the `CrashBackoff` loop will not occur."